### ISSUE TITLE
Drop-down to switch between documentation page versions includes versions where the page doesn't exist

### ISSUE THREAD

**Issue by Pinaksh** [fullcalendar/fullcalendar#4714]
### Bug Reports
Place - Full calendar documentation, just after edit document we have a drop-down list for the version selection,  While trying to switch the full calendar documentation from version 4.x to 3.x lot of times it does not do anything then we need to refresh page and need to select it again.

**Comment by acerix** [comment]
Do you mean after you switch the version to 3, then leave the page and come back, you would like it to show you v3 by default? (eg. remember by browser cookie)

**Comment by Pinaksh** [comment]
No no, not like that, suppose I am checking the documentation for any event, by default system shows 4.x version doc, but I am using 3.x calendar to trying to switch from drop down but unable to make selection lot of times. It may be you do not have documentation available for the 3.x for same event may be I am not sure or may be some JS issue while doing it.

**Comment by acerix** [comment]
I understand the issue now.  Switching between versions works for settings that have exactly the same name, but for example, [clientEvents](https://fullcalendar.io/docs/v3/clientEvents) was (sort of) replaced by [Calendar-getEvents](https://fullcalendar.io/docs/Calendar-getEvents), and both those pages have a drop-down to change the version, but changing between 3 and 4 doesn't work.  So, the drop-down options should only show when there actually is a doc page for that version.

**Comment by Pinaksh** [comment]
Yes, If documentation does not exists for the same event either dropdown options should be removed or redirect to the respective content. Thanks!

**Comment by dakala** [comment]
Could we have specific page URL(s) where this has been noticed please? Thanks.

**Comment by dakala** [comment]
I think this is the expected behaviour unless there are examples of where it doesn't work as expected. Examples are outlined below:

**All versions:**

The [render](https://fullcalendar.io/docs/render) method is vital for initializing a calendar. The version-selection dropdown is active and works for v1, v2, v3 and v4.

**v4.x only:**

[Initializing FullCalendar with an ES6-compatible build system](https://fullcalendar.io/docs/initialize-es6) like Webpack or Rollup along with a package manager like NPM or Yarn is only compatible with FullCalendar v4. Therefore, there's no documentation for other versions. The version-selection dropdown is greyed out for other versions except v4. 

**No v4.x:**

The [clientEvents](https://fullcalendar.io/docs/v3/clientEvents) method in earlier versions has no documentation available for v4. The dropdown shows v4 greyed out to show this. 

In summary, v1, v2, v3, v4 will be greyed out if there's no documentation available for a version of the current page.
